ui/page/start_page.go

```diff
@@ -413,3 +557,7 @@ func (sp *startPage) setLanguageSetting() {
 	}
 	values.SetUserLanguage(langPre)
 }
+
+// func (sp *startPage) recommendedSettings() {
+// 	To be implemented after settingsPage refactor
+// }
```

Comment:
```suggestion

// func (sp *startPage) recommendedSettings() {
// 	TODO: To be implemented after settings page refactor
// }
```

the function needs a comment to say what it does or it can be removed all together